src/lib/features/scheduler/scheduleServices.ts

```diff
@@ -0,0 +1,147 @@
+import {
+    hoursToMilliseconds,
+    minutesToMilliseconds,
+    secondsToMilliseconds,
+} from 'date-fns';
+import { IUnleashServices } from '../../server-impl';
+
```

Comment:
Would like to see a comment here

Reply:
Explored creating a custom linting rule in [Biome](https://github.com/biomejs/biome) and, as far as I could tell, I don't think that's supported yet. Added a comment in https://github.com/Unleash/unleash/pull/5363/commits/dcaaefd00058579bd2546cfe7990f1c1eec05d59